bitcoin/src/psbt/error.rs

```diff
@@ -80,6 +80,16 @@ pub enum Error {
     NegativeFee,
     /// Integer overflow in fee calculation
     FeeOverflow,
+    /// Non-witness UTXO (which is a complete transaction) has txid that
```

Comment:
nit - `txid` is lower case here while the acronym is upper cased elsewhere.

Reply:
Its like that all over the place, we could have an issue to fix it though.
```bash
 gg txid | grep ///
```